app/sequencescape_excel/sequencescape_excel/specialised_field/priority.rb

```diff
@@ -13,7 +13,6 @@ def update(_attributes = {})
         return unless valid?
 
         sample.priority = value if value.present?
-        sample.save
```

Comment:
Or possibly this one. I forget which the tests actually detected. But the other got missed as it depends on which templates are being used. Hence the more robust checks to ensure we don't accidentally re-introduce the bug.